src/test/java/com/example/rets_api/utils/CompareEntitiesTest.java

```diff
@@ -0,0 +1,122 @@
+package com.example.rets_api.utils;
+
+import com.example.rets_api.entity.*;
+
+public class CompareEntitiesTest {
```

Comment:
Please the name to CompareEntitiesUtilsTest because this is a Util class and not a test class